.github/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## Changelog
 
-## [1.7.0]
+### [1.7.4]
```

Comment:
You can just put this under 1.7.0 since we haven't actually released yet

Reply:
fixed